product-service-new/src/handlers/getProductsById.ts

```diff
@@ -0,0 +1,38 @@
+import books from './books.json'
+import { APIGatewayProxyHandler } from 'aws-lambda'
+
+export const getProductsById: APIGatewayProxyHandler = async (event) => {
+	try{
+		const { id } = event.pathParameters
+		const BookFound = books.find(({ id: filmID }) => filmID === id)
+		
+		if(!BookFound) {
+			return await{
```

Comment:
What the reason to use `await` here? Is it only for `use async/await` task? It is redundant coz no you have no any async operation.

Reply:
it was in the task and it's not redundant for future operations and extentions